src/service/performance-impl.js

```diff
@@ -691,6 +694,15 @@ export class Performance {
       data['value'] = this.timeOrigin_ + delta;
     }
 
+    // Emit events. Used by `gulp performance`.
+    this.win.dispatchEvent(
```

Comment:
This might actually lead to a perf regression. https://github.com/ampproject/amphtml/issues/22975 noticed that firing the `BUILT` event for each amp element was causing a noticeable delay in FCP. It depends on how often this is firing, which could be very frequently.

Reply:
As discussed in chat, it fires < 30 times per page load. That said, I'll look into changing it to a top-level function like `AMP.getPerfStats` instead.